2011-01-19  Michael Tautschnig  <mt@debian.org>

	* simple example: Added class GRUB_PC that installs and uses grub-pc instead
		of grub
Index: trunk/examples/simple/package_config/DEBIAN
===================================================================
--- trunk.orig/examples/simple/package_config/DEBIAN
+++ trunk/examples/simple/package_config/DEBIAN	
@@ -17,10 +17,10 @@
 dhcp3-client
 
 PACKAGES aptitude GRUB
-grub-legacy lilo-
+grub-legacy grub-pc- lilo-
 
 PACKAGES aptitude GRUB_PC
 grub-pc grub-legacy- lilo-
 
 PACKAGES aptitude LILO
-lilo grub-
+lilo grub-legacy- grub-pc-
Index: trunk/examples/simple/class/50-host-classes
===================================================================
--- trunk.orig/examples/simple/class/50-host-classes
+++ trunk/examples/simple/class/50-host-classes	
@@ -20,5 +20,4 @@
 	echo "FAIBASE DEBIAN DHCPC" ;;
 esac
 
-(ifclass I386 || ifclass AMD64) && echo GRUB 
 exit 0
Index: trunk/examples/simple/class/51-grub
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ trunk/examples/simple/class/51-grub	
@@ -0,0 +1,4 @@
+#! /bin/bash
+
+{ ifclass I386 || ifclass AMD64; } && ! ifclass GRUB_PC && echo GRUB
+exit 0
